XamlControlsGallery/ControlPages/ButtonPage.xaml

```diff
@@ -58,6 +58,45 @@
                 </x:String>
             </local:ControlExample.Xaml>
         </local:ControlExample>
+        <local:ControlExample HeaderText="Wrapping Buttons with large content">
+            <StackPanel MaxWidth="500">
+                <TextBlock Text="The following buttons have content that would get clipped if we are not wrapping the buttons properly." Margin="0,0,0,8" TextWrapping="Wrap"/>
+                <TextBlock Text="The first option to mitigate cut of content is to place them underneath each other:" Margin="0,0,0,8" TextWrapping="Wrap"/>
+                <Button HorizontalAlignment="Stretch" Margin="0,0,0,5">This is some text that is too long and will get cut off</Button>
+                <Button HorizontalAlignment="Stretch">This is another text that would result in being cut off</Button>
+
+                <TextBlock Text="Another option is to increase the height of the buttons:" Margin="0,8,0,8"/>
+                <StackPanel Orientation="Horizontal" HorizontalAlignment="Center">
+                    <Button MaxWidth="240" Margin="0,0,8,0" Height="60">
```

Comment:
You actually don't want to hardcode the Height. With large, this will still get cut off:
![image](https://user-images.githubusercontent.com/13142665/97760634-d684c300-1ac0-11eb-9d0b-5c999234cac9.png)

If you need a height, use MinHeight. Or you can simply leave off that property and the Button (inside a Horizontal StackPanel, at least) will adjust automatically.

Reply:
Same here, great suggestion, fixed that now.